awx/api/serializers.py

```diff
@@ -4767,6 +4768,28 @@ def validate(self, attrs):
         return super(ScheduleSerializer, self).validate(attrs)
 
 
+class InstanceLinkSerializer(BaseSerializer):
+    class Meta:
+        model = InstanceLink
+        fields = ('source', 'target')
+
+    source = serializers.SlugRelatedField(slug_field="hostname", read_only=True)
+    target = serializers.SlugRelatedField(slug_field="hostname", read_only=True)
```

Comment:
I don't see any other uses of this field type in the file. What do the OPTIONS for the endpoint using this look like?

Reply:
well I guess OPTIONS won't be useful for anything, since it's not employed in a real list view.

Reply:
OPTIONS for this end point look exactly the same as OPTIONS for `/job_templates/labels`